src/Node.php

```diff
@@ -123,8 +123,11 @@ public function __wakeup()
     public function getLdap()
     {
         if ($this->ldap === null) {
-            throw new Exception\LdapException(null, 'No LDAP connection specified.',
-                Exception\LdapException::LDAP_OTHER);
+            throw new Exception\LdapException(
+                null,
+                'No LDAP connection specified.',
+                Exception\LdapException::LDAP_OTHER
+            );
```

Comment:
These fixes were because phpcs was giving me grief about multiple arguments. I figured I'd go ahead and fix them.